src/entities/on_ramp/index.ts

```diff
@@ -7,14 +8,16 @@ const OnRamp_Entities = [
     RampOrderModel,
     RampTransfer,
     TransakWebHookEvent,
-    OnrampOrderAccounting
+    OnrampOrderAccounting,
+    onRampOrder
```

Comment:
make it OnRampOrder to keep it consistent

Reply:
I had changed